tests/unittests/core/conftest.py

```diff
@@ -155,20 +156,166 @@ def random_dt(monkeypatch):
+        "start_time": None,
+        "end_time": None,
+        "results": [{"name": None, "type": "objective", "value": None}],
+        "params": [
+            {"name": "/decoding_layer", "type": "categorical", "value": "rnn"},
+            {"name": "/encoding_layer", "type": "categorical", "value": "gru"},
+        ],
+        "parents": [],
+    },
+    {
+        "experiment": "supernaedo2-dendi",
+        "status": "new",
+        "worker": None,
+        "submit_time": MockDatetime(2017, 11, 23, 2),
+        "start_time": None,
+        "end_time": None,
+        "results": [{"name": None, "type": "objective", "value": None}],
+        "params": [
+            {
+                "name": "/decoding_layer",
+                "type": "categorical",
+                "value": "lstm_with_attention",
+            },
+            {"name": "/encoding_layer", "type": "categorical", "value": "rnn"},
+        ],
+        "parents": [],
+    },
+    {
+        "experiment": "supernaedo2-dendi",
+        "status": "interrupted",
+        "worker": None,
+        "submit_time": MockDatetime(2017, 11, 23, 3),
+        "start_time": MockDatetime(2017, 11, 23, 3),
+        "end_time": None,
+        "results": [{"name": None, "type": "objective", "value": None}],
+        "params": [
+            {
+                "name": "/decoding_layer",
+                "type": "categorical",
+                "value": "lstm_with_attention",
+            },
+            {"name": "/encoding_layer", "type": "categorical", "value": "lstm"},
+        ],
+        "parents": [],
+    },
+    {
+        "experiment": "supernaedo2-dendi",
+        "status": "suspended",
+        "worker": None,
+        "submit_time": MockDatetime(2017, 11, 23, 4),
+        "start_time": MockDatetime(2017, 11, 23, 4),
+        "end_time": None,
+        "results": [{"name": None, "type": "objective", "value": None}],
+        "params": [
+            {"name": "/decoding_layer", "type": "categorical", "value": "gru"},
+            {"name": "/encoding_layer", "type": "categorical", "value": "lstm"},
+        ],
+        "parents": [],
+    },
+]
+
+
 @pytest.fixture()
-def hacked_exp(with_user_dendi, random_dt, clean_db, create_db_instance):
-    """Return an `Experiment` instance with hacked _id to find trials in
-    fake database.
-    """
-    exp = experiment_builder.build(name="supernaedo2-dendi")
-    exp._id = "supernaedo2-dendi"  # white box hack
-    return exp
+def hacked_exp(with_user_dendi, random_dt, storage):
+    """Return an `Experiment` instance to find trials in fake database."""
+    with OrionState(experiments=[dendi_exp_config], trials=dendi_base_trials) as cfg:
+        exp = experiment_builder.build(name=dendi_exp_config["name"])
+        exp._id = cfg.trials[0]["experiment"]
+        yield exp
 
 
 @pytest.fixture()
-def trial_id_substitution(with_user_tsirif, random_dt, clean_db, create_db_instance):
+def trial_id_substitution(with_user_tsirif, random_dt, pdatabase):
```

Comment:
You can remove both substitution fixtures.

Reply:
What do you mean?

Reply:
This fixture and the one below can be removed. There are only needed for the skipped tests of experiment_tree. Actually, let's nuke these tests. :grin:

Reply:
I nuked them 💥